Dockerfile

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+WORKDIR /src
+
+COPY . /src/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        bsd-compat-headers \
+        bzip2-dev \
+        cmake \
+        curl-dev \
+        file \
+        fts-dev \
+        g++ \
+        git \
+        json-c-dev \
+        libmilter-dev \
+        libtool \
+        libxml2-dev \
+        linux-headers \
+        make \
+        ncurses-dev \
+        openssl-dev \
+        pcre2-dev \
+        python3 \
+        zlib-dev \
+    && \
+    mkdir -p "./build" && \
+    cd "./build" && \
+    cmake .. \
+          -DAPP_CONFIG_DIRECTORY="/etc/clamav" \
+          -DCMAKE_BUILD_TYPE"=release" \
+          -DCMAKE_INSTALL_FULL_BINDIR="/usr/bin" \
+          -DCMAKE_INSTALL_FULL_INCLUDEDIR="/usr/include" \
+          -DCMAKE_INSTALL_FULL_LIBDIR="/usr/lib" \
+          -DCMAKE_INSTALL_FULL_SBINDIR="/usr/sbin" \
+          -DCMAKE_INSTALL_LIBDIR="/usr/lib" \
+          -DCMAKE_INSTALL_PREFIX="/usr" \
+          -DENABLE_CLAMONACC=OFF \
+          -DENABLE_DOCS=OFF \
```

Comment:
Recommend removing this line so we can provide manpages

Reply:
it is very uncommon to have manpages in the container however. How do you envision this? It's not like users will open the container to read the manpage ... I've never seen it either :) i'm pondering how to make this work technically from the docker context as well .. `docker exec -it <running-container> /bin/sh`; and then executing man might work but its a bid crudgy.
I guess that less than 0.01% of the container users would use this is my guestimate ...

Reply:
I often do something like `docker run -it <image tag> /bin/sh` to test in a container. But I see your point - most folks set an entry point for their docker images and run it in the background. And since most docker images lack manpages, it's unlikely anyone will complain that they're missing.

For the Snap, I made an incorrect assumption that classic mode snaps could have manpages. It sounds like manpages aren't supported for CLI snaps (https://forum.snapcraft.io/t/support-for-man-pages/2299/21).  😞